management_interface/management_interface/models.py

```diff
@@ -0,0 +1,56 @@
+import hashlib
+
+from django.db import models
+import uuid
+from datetime import datetime
+
+
+class BaseModel(models.Model):
+    """
+    Base model holding shared data
+    """
+    id = models.CharField(
+        primary_key=True, max_length=64, default=uuid.uuid4, editable=False, unique=True
+    )
+    created_at = models.DateTimeField(default=datetime.utcnow, editable=False)
+    updated_at = models.DateTimeField(auto_now=True, editable=False)
+
+
+class CareProviderLocation(BaseModel):
+    """
+    Model that represents a registered care provider location (branch) who wants to receive emails.
+    """
+    name = models.CharField(max_length=256, help_text="Helping Hands Wantage")
+    email = models.EmailField(null=False, db_index=True, unique=True, help_text="example@nhs.net")
```

Comment:
Looks like Henry thought about the same thing. The location can be different but the provider might be the same. I remove this constraint, it's not necessary but might cause issues later.

Reply:
It's a business process constraint I'm not sure we've worked out yet - do we actually want to allow Care Provider Locations to have a non-unique email. 

I think this might make it harder to trace back to who we've sent it to and not sure whether we want to allow other locations to view info on those receipients.

Reply:
In any case we should probably relax the model to remove the unique flag

Reply:
Resolving on the basis that it's easy enough to change later, if that's what we decide to do.